tests/resources/Vsphere-Util.robot

```diff
@@ -36,6 +36,8 @@ Power Off VM OOB
 
 Destroy VM OOB
     [Arguments]  ${vm}
+    ${rc}  ${output}=  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Run And Return Rc And Output  govc object.method -name Destroy_Task -enable %{VCH-NAME}/${vm}
```

Comment:
You need to use the full inventory path with the object.* commands: `/dc-name/vm/vapp-name/vm-name` 
Of if there is a single datacenter, it can be relative to that: `vm/vapp-name/vm-name`